templates/CRM/Admin/Form/Preferences/Address.tpl

```diff
@@ -18,7 +18,10 @@
                 {help id="id-token-text" tplFile=$tplFile file="CRM/Contact/Form/Task/Email.hlp"}
               </div>
                 {$form.mailing_format.html|crmAddClass:huge12}<br />
-                <span class="description">{ts}Content and format for mailing labels.{/ts}</span>
+                <span class="description">{ts}Content and format for mailing labels.{/ts}<br />
+                  {capture assign=labelFormats}{crmURL p='civicrm/admin/labelFormats' q="reset=1"}{/capture}
+                  {ts 1=$labelFormats}You can change the size and layout of labels at <a href="%1">Label Page Formats</a>.{/ts}
```

Comment:
I know there's some pre-existing below but the best practice is to do `<a %1>` since then the params can change without requiring retranslating: https://docs.civicrm.org/dev/en/latest/translation/#avoid-tags-inside-strings (where it says hyperlinks).

So `{ts 1='href="$labelformats" target="_blank"'}... <a %1>Label Page Formats</a>.{/ts}`

Reply:
OK, will fix these. Do we prefer:
```
{capture assign=labelFormats}{crmURL p='civicrm/admin/labelFormats' q="reset=1"}{/capture}
{ts 1="href=`$labelFormats`"}
 ```
or
```
{capture assign=labelFormats}href="{crmURL p='civicrm/admin/labelFormats' q="reset=1"}"{/capture}
{ts 1=$labelFormats}
 ```

Reply:
Oh wait, the first one of those is no good, no quotes. So option B?
Smarty won't evaluate within single quotes is the issue.

Reply:
Nice quote-hell. Yeah I guess the second one, and then that works too if there's also more params like `target="_blank"`.
Maybe add a `{* code comment *}` that it's gnarly because of quote issues.

Reply:
Changed these and pushed a new version.
Also added [this to the docs](https://lab.civicrm.org/documentation/docs/dev/-/merge_requests/1101) (confirmed that it is the right way in other examples).